app/pubspec.yaml

```diff
@@ -22,6 +22,7 @@ dependencies:
   flip_card: ^0.7.0
   flutter:
     sdk: flutter
+  flutter_tts: ^4.2.2
```

Comment:
There should be empty line  below `flutter:` like there was before.
It is a lint thing.